docs/UserGuide.md

```diff
@@ -127,6 +157,18 @@ Format: `archive`
 * A copy of the details currently saved in the address book will be saved to a separate file.
 * The file name will be the date and time of the archive operation.
 
+### Displaying manual for a command: `manual`
+
+Display the format for a specified command and a short description for that command.
+
+Format: `manual COMMAND_NAME`
+
+* The format for the command corresponding to the command name will be displayed, along with a short description
```

Comment:
```suggestion
* The format for the command corresponding to the command name will be displayed, along with a short description.
```

Reply:
Might want to include what the user should expect if an invalid command is supplied to `manual` (maybe an error message or smt?)